internal/provider/resources/automation_test.go

```diff
@@ -322,8 +322,8 @@ func TestAccResource_automation(t *testing.T) {
 					resource.TestCheckResourceAttr(metricTriggerAutomationResourceNameAndPath, "name", "test-metric-automation"),
 					resource.TestCheckResourceAttr(metricTriggerAutomationResourceNameAndPath, "description", "description for test-metric-automation"),
 					resource.TestCheckResourceAttr(metricTriggerAutomationResourceNameAndPath, "enabled", "true"),
-					testutils.TestCheckJSONAttr(metricTriggerAutomationResourceNameAndPath, "trigger.metric.match", `{"prefect.resource.id":"prefect.flow-run.*"}`),
-					testutils.TestCheckJSONAttr(metricTriggerAutomationResourceNameAndPath, "trigger.metric.match_related", `{"prefect.resource.id":"prefect.flow-run.*","prefect.resource.role":"flow"}`),
+					resource.TestCheckResourceAttr(metricTriggerAutomationResourceNameAndPath, "trigger.metric.match", testutils.NormalizedValueForJSON(t, `{"prefect.resource.id":"prefect.flow-run.*"}`)),
+					resource.TestCheckResourceAttr(metricTriggerAutomationResourceNameAndPath, "trigger.metric.match_related", testutils.NormalizedValueForJSON(t, `{"prefect.resource.id":"prefect.flow-run.*","prefect.resource.role":"flow"}`)),
```

Comment:
The changes to this automations test file use the `NormalizedValueForJSON` function added in this PR, which allowed me to remove the `TestCheckJSONAttr` function for consistency. I'm happy to restore that, though, if the existing setup was more clear (b5fc151).